tree/ntuple/v7/src/RField.cxx

```diff
@@ -1747,6 +1753,125 @@ void ROOT::Experimental::RClassField::AcceptVisitor(Detail::RFieldVisitor &visit
 
 //------------------------------------------------------------------------------
 
+std::size_t ROOT::Experimental::RField<TObject>::GetOffsetOfMember(const char *name)
+{
+   if (auto dataMember = TObject::Class()->GetDataMember(name)) {
+      return dataMember->GetOffset();
+   }
+   throw RException(R__FAIL('\'' + std::string(name) + '\'' + " is an invalid data member"));
+}
+
+ROOT::Experimental::RField<TObject>::RField(std::string_view fieldName)
+   : ROOT::Experimental::RFieldBase(fieldName, "TObject", ENTupleStructure::kRecord, false /* isSimple */)
+{
+   assert(TObject::Class()->GetClassVersion() == 1);
```

Comment:
```suggestion
   TClass *cl = TObject:Class();
   assert(cl->GetClassVersion() == 1);
```
`::Class()` is not "super fast" (at least a function call and an atomic load) and this "also" save some characters :)

Reply:
Since the pointer is only use once now, the comment is moot.